news/2019-04-24-TWC-2019-Group-Stage-Recap.md

```diff
@@ -0,0 +1,70 @@
+
+**Conversely, [Shano x 40mP - Sanpo Michi](https://osu.ppy.sh/beatmapsets/62991#taiko/186452) holds the title of most banned map of the week, with 18 bans to 17 picks.** Understandably, it was clear that many teams didn’t want to risk tackling the devilishly tricky 120BPM map on Hidden. Finally, [CyoucyoP feat.Hatsune Miku - End of the World](https://osu.ppy.sh/beatmapsets/215476#taiko/506018) received the least attention at 16 interactions - 10 picks and 6 bans. A bit disappointing, but the map has made more than its fair share of appearances in past TWCs - it’s certainly not the end of the world as the song would otherwise suggest.
+
+--------
+## <a name="matches" id="matches"></a>Tournament Matches
+
+
+
+### Saturday:
+
+
+### Sunday:
+
+--------
+If you missed out on any of the action last weekend, you can always watch the VODs on the [osu!live Twitch Channel](https://www.twitch.tv/osulive/videos).
+
+That concludes all the matches for now! Let's wave our goodbyes to the 16 teams that were narrowly defeated last weekend - we thank you for participating and hope to see you again next year. Good luck to all the remaining teams in this tournament!
+
+--------
+## <a name="ahead" id="ahead"></a>Looking Ahead: Round of 16
+
+**Next week we'll be treated to a total of 8 exciting matches, and you can find the schedule on the [TWC 2019 wiki](https://osu.ppy.sh/help/wiki/Tournaments/TWC/2019#match-schedule:-round-of-16).** As it is a double elimination format, the Round of 16 is unique in the sense that no team is at risk of being eliminated, though their performance will determine how the remainder of the matches play out in future stages. 
+
+**The first match of the RO16 Stage will take place Saturday 11:00 UTC, 27th of April.** There we will see a much anticipated match between two veteran countries - Japan vs. France. Be sure to catch the stream on the [osu!live twitch channel!](https://www.twitch.tv/osulive). 
+
+**Finally, a reminder to buy your banners today!** In addition to expressing your support for a chosen team, your contribution will also go towards funding the prize pool for this tournament. You can purchase the banners [here](https://osu.ppy.sh/store/products/ABC).
+
+--------
+And that's all from me this week! I’d like to take this opportunity to thank [PuffBuck](https://osu.ppy.sh/users/4234525) for his help on collating information. I look forward to working closely with him in the coming weeks. If you have any feedback or suggestions about the format of these recaps, feel free to drop me a PM, or leave a comment down below. Otherwise, see you next week!
+
+Remember, rhythm is only a click away!
```

Comment:
probably not needed, you already said goodbye

Reply:
It's a reminder, not a goodbye.